core/src/test/java/dk/alexandra/fresco/framework/util/TestAesCtrDrbg.java

```diff
@@ -5,10 +5,25 @@
 
 import java.util.Arrays;
 import java.util.Random;
+import org.hamcrest.core.IsEqual;
+import org.hamcrest.core.IsNot;
+import org.junit.Assert;
```

Comment:
I prefer static imports for these assertions, but really it is a matter of taste.